classes/items/itembase.hxx

```diff
@@ -18,6 +18,7 @@ class Item {
 
 //Private variables
 private:
+    unsigned short id;
```

Comment:
need to also have a static property next_id that's also an unsigned short and starts from 0.